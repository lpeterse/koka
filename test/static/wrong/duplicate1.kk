alias dup = int
alias foo = dup
alias bar = foo
alias dup = list
