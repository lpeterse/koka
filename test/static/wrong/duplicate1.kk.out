test/static/wrong/duplicate1.kk(4, 1): error: Type static/wrong/duplicate1/dup is already defined